src/tests.rs

```diff
@@ -610,6 +625,44 @@ fn cannot_remove_supersig() {
 	})
 }
 
+#[test]
+fn cannot_liquidate_supersig() {
+	ExtBuilder::default().balances(vec![]).build().execute_with(|| {
+		assert_ok!(Supersig::create_supersig(
+			Origin::signed(ALICE()),
+			vec!(ALICE(), BOB(), CHARLIE()),
+		));
+		let supersig_id = get_account_id(0);
+
+		let call = Call::Balances(pallet_balances::Call::transfer_all {
+			dest: ALICE(),
+			keep_alive: false,
+		});
+
+		assert_ok!(Supersig::submit_call(
+			Origin::signed(ALICE()),
+			supersig_id.clone(),
+			Box::new(call.clone())
+		));
+
+		assert_ok!(Supersig::approve_call(
+			Origin::signed(BOB()),
+			supersig_id.clone(),
+			0
+		));
+
+		assert_ok!(Supersig::approve_call(
+			Origin::signed(CHARLIE()),
+			supersig_id.clone(),
+			0
+		));
+
+		assert!(Supersig::calls(0, 0).is_none());
+
+		assert!(System::account_exists(&supersig_id));
```

Comment:
The supersig account was not liquidated due to the consumer counter being 1. Despite `keep_alive: false`. If you remove the line where I increment the rc for the supersig in create account, this test will fail